girder/models/folder.py

```diff
@@ -309,7 +309,6 @@ def remove(self, folder, progress=None, **kwargs):
             if progress:
                 progress.update(increment=1, message='Deleted item ' +
                                 item['name'])
-        items.close()
```

Comment:
That's understandable. I'll change this to either comment on the reason for the close, or refactor to make it automatically close sooner (I need to see if this is possible).